plugins/es.upv.paella.playPauseButtonPlugin/playbutton_onscreen.js

```diff
@@ -103,6 +103,7 @@ paella.addPlugin(function() {
 	
 		endVideo() {
 			this.isPlaying = false;
+                        this.showIcon = true;
 			this.checkStatus();
```

Comment:
Sorry, I'll fix that extra indent. I was thinking it was just from the "+" , but it's actually there.